filebeat/crawler/crawler.go

```diff
@@ -44,7 +44,7 @@ func (c *Crawler) Start(r *registrar.Registrar, reloaderConfig *common.Config) e
 	}
 
 	if reloaderConfig.Enabled() {
-		logp.Warn("EXPERIMENTAL feature dynamic configuration reloading is enabled.")
+		logp.Warn("BETA feature dynamic configuration reloading is enabled.")
```

Comment:
I think it still fits the Beta description. We did have *Known issues* in the release notes in the past, for example: https://www.elastic.co/guide/en/beats/libbeat/current/release-notes-5.0.0-alpha1.html#_known_issues